packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -1,273 +1,375 @@
+                <Li>
+                  {_.keyValue(
+                    _('compression'),
+                    compression === 'native' ? 'GZIP' : compression
+                  )}
+                </Li>
+              )}
+            </Ul>
+          )
+        },
+        name: _('formNotes'),
+      },
+    ],
+    individualActions: [
+      {
+        handler: (job, { goTo }) =>
+          goTo({
+            pathname: '/home',
+            query: { t: 'VM', s: constructQueryString(job.vms) },
+          }),
+        disabled: job => job.type !== 'backup',
+        label: _('redirectToMatchingVms'),
+        icon: 'preview',
+      },
+      {
+        handler: (job, { goTo }) => goTo(`/backup/${job.id}/edit`),
+        label: _('formEdit'),
+        icon: 'edit',
+        level: 'primary',
+      },
+    ],
   }
 
-  _getScheduleCollection = createSelector(
-    this._getSchedules,
-    () => this.props.jobs,
-    (schedules, jobs) => {
-      if (!schedules || !jobs) {
-        return []
-      }
-
-      return map(schedules, schedule => {
-        const job = jobs[schedule.jobId]
-        const { items } = job.paramsVector
-        const pattern = get(items, '[1].collection.pattern')
-
-        return {
-          jobId: job.id,
-          jobLabel: jobKeyToLabel[job.key] || _('unknownSchedule'),
-          redirect:
-            pattern !== undefined &&
-            (() => this._redirectToMatchingVms(pattern)),
-          // Old versions of XenOrchestra use items[0]
-          scheduleTag:
-            get(items, '[0].values[0].tag') ||
-            get(items, '[1].values[0].tag') ||
-            schedule.id,
-          schedule,
-        }
-      })
-    }
-  )
+  _goTo = path => {
+    this.context.router.push(path)
+  }
 
-  _getIsScheduleUserMissing = createSelector(
-    this._getSchedules,
+  _getCollection = createSelector(
     () => this.props.jobs,
-    () => this.props.users,
-    (schedules, jobs, users) => {
-      const isScheduleUserMissing = {}
-      forEach(schedules, schedule => {
-        isScheduleUserMissing[schedule.id] = !(
-          jobs && find(users, user => user.id === jobs[schedule.jobId].userId)
-        )
-      })
-
-      return isScheduleUserMissing
-    }
+    () => this.props.metadataJobs,
+    (jobs = [], metadataJobs = []) => [...jobs, ...metadataJobs]
   )
 
   render() {
-    const schedules = this._getSchedules()
-    const isScheduleUserMissing = this._getIsScheduleUserMissing()
-
     return (
-      <div>
+      <SortedTable
+        {...JobsTable.tableProps}
+        collection={this._getCollection()}
+        data-goTo={this._goTo}
+        data-schedulesByJob={this.props.schedulesByJob}
+      />
+    )
+  }
+}
+
+const LegacyJobKeyToLabel = {
+  continuousReplication: _('continuousReplication'),
+  deltaBackup: _('deltaBackup'),
+  disasterRecovery: _('disasterRecovery'),
+  rollingBackup: _('backup'),
+  rollingSnapshot: _('rollingSnapshot'),
+}
```

Comment:
Why do you need this?

Reply:
To check that the legacy `job` is a `backup`.

Reply:
But why do you need the whole map? Don't you only need a set of keys?